cfme/infrastructure/config_management.py

```diff
@@ -294,12 +294,16 @@ def refresh_relationships(self, cancel=False):
         if not cancel:
             flash.assert_message_match(self._refresh_flash_msg)
 
+    def does_profile_exist(self):
```

Comment:
From now on, methods that do not navigate should be prefixed with a "_" so tha they are not intended for general use. Once this is done I am happy to merge this PR